src/core/lib/viewport-transition-utils.js

```diff
@@ -90,6 +92,12 @@ export function viewportLinearInterpolator(startViewport, endViewport, t) {
 export function viewportFlyToInterpolator(startViewport, endViewport, t) {
   // Equations from above paper are referred where needed.
 
+  // Assert minimum required props
+  for (const key of ['latitude', 'longitude', 'zoom', 'width', 'height']) {
+    Number.isFinite(startViewport[key]);
+    Number.isFinite(endViewport[key]);
```

Comment:
This is not doing anything. Missing `assert`?

Reply:
Yes, missing assert.